src/content/desci/index.md

```diff
@@ -8,77 +8,268 @@ sidebar: true
+
+By creating knowledge that is impartial and available to the public, we enable ourselves to make fundamental technological advancements. These advancements give us the power to more effectively navigate and manipulate our environment, leading to vast improvements in quality of life. As we continue to create impartial knowledge and make technological advancements, we find ourselves capable of managing increasingly complex systems. Today, we are confronted with species-level existential risks, such as climate change and global pandemics. Science is the only process we have that offers equitable positive outcomes.
+
+
+## How DeSci improves science
+
+**List of key problems in science and how decentralized science can help to address these issues**
+
+
+
+* bureaucratic funding structures -> allow for more diverse funding sources (quadratic donations, DAOs, crowdfunding etc.) 
+* replication crisis -> allowing for easier access to data and methods, and by providing incentives for reproducibility.
+* misaligned incentives -> aligning incentives more closely with the goals of science. 
+* poor data quality -> incentivising transparent data sharing and quality control. 
+* publication bias -> enabling publication in a wider range of outlets. 
+* p-hacking -> incentivising transparent and honest reporting of results.
+* cherry picking -> requiring publication of all results, including negative or inconclusive ones. 
+* HARKing: hypothesizing after the results are known -> providing incentives for transparent and honest reporting of results.
+
+A comparison to traditional science
+
+
+<table>
+  <tr>
+   <td><strong>DeSci</strong>
+   </td>
+   <td><strong>Traditional science</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>You raise funding from a global community.
+   </td>
+   <td>You raise funding from a national agency.
+   </td>
+  </tr>
+  <tr>
+   <td>You collaborate with peers from all over the globe in dynamic teams.
+   </td>
+   <td>Your collaborations are limited by funding organizations and home institutions.
+   </td>
+  </tr>
+  <tr>
+   <td>Funding decisions are made online and in a transparent way. New funding mechanisms are explored.
+   </td>
+   <td>Funding decisions are made with a long turnaround time and limited transparency. Few funding mechanisms exist.
+   </td>
+  </tr>
+  <tr>
+   <td>DeSci is open to anyone.
+   </td>
+   <td>You must be a member or a physical research institute to do science.
+   </td>
+  </tr>
+  <tr>
+   <td>You can trade laboratory services freely.
+   </td>
+   <td>You are constrained to using your local institute's laboratory infrastructure.
```

Comment:
again, I'm not sure this is really true - laboratory and computing resources are shared between institutions all the time. If there is a barrier it is usually monetary.